### PR TITLE
fix: SearchBar autofocus when click on clear btn

### PR DESCRIPTION
**Pull request by MrSIHAB**
The search bar should automatically focused when 'clear search text' button is clicked. This is small tweak but usefull too. I do search, do some activities. Then search bar become unfocused. Then I click on clear search button and start typing my query, And then I see that Search bar is not focused. I have to manually click on the textfeild to focus it. This pull request will automatically focus search bar when someone click on clear search button. I made this changes from my experience.

### CODE REVIEW

File: packages/app_center/lib/search/search_field.dart [hunk 1]
```diff
@@ -188,7 +188,10 @@ class _SearchFieldState extends ConsumerState<SearchField> {
                       return YaruIconButton(
                         icon: const Icon(YaruIcons.edit_clear, size: 16),
                         onPressed:
-                            controller.text.isEmpty ? null : controller.clear,
+                            controller.text.isEmpty ? null : (){
+                              controller.clear();
+                              node.requestFocus();
+                        },
```
Comment (spydon) [inline packages/app_center/lib/search/search_field.dart:194]: I think it should always focus the field, even if there isn't any text in it.

**Review by spydon — COMMENTED** [review]
Run `dart format` to format the code properly. :)

**Review by spydon — APPROVED** [review]
I fixed up the formatting, so now it looks shippable!

**Comment by MrSIHAB** [comment]
Thanks. Next time I'll take care of it

**Pull request merged**